Server/src/models/DatasetModels/DatasetApprovalModel.ts

```diff
@@ -63,6 +63,26 @@ export class DatasetApprovalModel {
         return "Dataset Flagged!"
     }
 
+    async verifyUnapprovedDatasetUploader(id: number, userId: number): Promise<any> {
+        let upload = await this.connection.createQueryBuilder(Unapproveddatasets, 'unapproved_datasets')
+            .select('dataset.uploaderId', 'uploaderId')
+            .addSelect('unapproved_datasets.isFlagged', 'isFlagged')
+            .innerJoin(Dataset, 'dataset', 'dataset.id = unapproved_datasets.datasetId')
+            .where('dataset.id = :id', { id: id })
+            .getRawOne();
+        console.log(userId)

```

Comment:
are the console logs necessary or just for debugging?

Reply:
Good catch; debugging and I missed removing those when I did a pass for logs. Removed

Reply:
good job